docs/tasks/administer-cluster/securing-a-cluster.md

```diff
@@ -0,0 +1,186 @@
+---
+assignees:
+- smarterclayton
+title: Securing a Cluster
+redirect_from:
+- "/docs/admin/cluster-security/"
+- "/docs/admin/cluster-security.html"
+- "/docs/concepts/cluster-administration/cluster-security/"
+- "/docs/concepts/cluster-administration/cluster-security.html"
+---
+
+* TOC
+{:toc}
+
+This document covers topics related to protecting a cluster from accidental or malicious access
+and provides recommendations on overall security.
+
+## Controlling access to the Kubernetes API
+
+As Kubernetes is entirely API driven, controlling and limiting who can access the cluster and what actions
+they are allowed to perform is the first line of defense.
+
+### Use Transport Level Security (TLS) for all API traffic
+
+Kubernetes expects that all API communication in the cluster is encrypted by default with TLS, and the
+majority of installation methods will allow the necessary certificates to be created and distributed to 
+the cluster components. Note that some components and installation methods may enable local ports over 
+HTTP and administrators should familiarize themselves with the settings of each component to identify 
+potentially unsecured traffic.
+
+### API Authentication
+
+When a cluster is installed, choose an appropriate authentication mechanism for the API servers to use that 
```

Comment:
[RW - clarity] Choose an authentication mechanism for the API servers to use that matches the common access patterns when you install a cluster.